### ISSUE TITLE
Tacotron2

### ISSUE THREAD

**Issue by StrongTanisha** [StrongResearch/isc-demos#50]
**Source / repo**

https://github.com/pytorch/audio/tree/main/examples/pipeline_tacotron2

**Model description**

[DESCRIPTION]

**Dataset**

LJ Speech Dataset
https://keithito.com/LJ-Speech-Dataset/

**Literature benchmark source**

[URL]

**Literature benchmark performance**

[DESCRIPTION]
[VALUE/S]

**Strong Compute result achieved**

[VALUE/S]

**Basic training config (as applicable)**

Nodes: 10
Epochs: 1500
Effective batch size: [N]
Learning rate: [L]
Optimizer: [OPT]

**Logs gist**

[URL]
